MetaMask/metamask-mobile
# [5.2][Fix] Favourites not showing when home button is pressed in browser tab menu

**Description**

Fix no favourites shown when "home" is pressed in the browser, scripts are now guaranteed to be present when the website is loaded.

This is blocked by https://github.com/MetaMask/dapps/pull/122 and should be tested and merged together.

**How to test**

- Make sure you check out both branches (app and dapp repo);
- On [this line](https://github.com/MetaMask/metamask-mobile/blob/fix/home-favorites/app/core/AppConstants.js#L32) the URL should match your app local URL either `http://localhost:3000/` or `http://${your-ip}:3000/`
- Also on [this line](https://github.com/MetaMask/metamask-mobile/blob/fix/home-favorites/app/components/Views/BrowserTab/index.js#L67) it should be changed accordingly like `localhost:3000` or `{your-ip}:3000`


**Checklist**


**Issue**

Progresses #3331
